data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	Through it all, Taely offers a running commentary on how each stretch of the ceremony represents a different piece of their collective history, ranging from the launch of the first human into space to its commercialization, and then the new age of exploration opened by the advent of hyperdrives. It also includes numerous conflicts, disasters, and wars. Much of it is familiar to you from what you know of human history, but the later part is strange and unfamiliar. Filled with stories of first contact with the aliens, discovery of strange ruins, alien invasions, desperate last stands, and brilliant discoveries, you are left with the impression that despite the Remnant's current security, they have been on the precipice of extinction for much of their existence.`
+			choice
+				`	(Stand still and absorb the moment.)`
+				`	(Look around to see what is happening next.)`
+			`	Around you, everyone stands still. Most look exhausted, and some are sitting on the floor or leaning against the wall. A few people look like they have been crying, with those nearby hugging them or touching their shoulder. Most of the people you can see have reverent and joyful expressions. After what feels like several minutes the doors open and people start to leave, and in short order the cave is nearly empty.`
+			label celebrationend
+			`	"I always find these Celebrations of Remembrance to be invigorating. What did you think of it, <first>?"`
+			choice
+				`	"I think I will have to get back to you on that later."`
+					goto celebrationrequest
+				`	"That was awe-inspiring."`
+					goto awestruck
+				`	"Why were some of the people crying?`
+					goto askcrying
+				`	"Who leads these ceremonies?"`
+					goto ceremonyleader
+				`	"That seemed both well-rehearsed and very disorganized."`
+			label celebrationrehearsed
+			`	Taely smiles at that. "Well, it is and it is not. We hold these several times a month, with most people taking part in one every few weeks. So most of us have attended hundreds of these. Although as far as that goes, every one of these celebrations is different. Each person's participation is up to them, and so the details and feel of each celebration are unique every time." She makes a gesture you have come to realize amounts to "If you have more questions, ask them."`
```

Comment:
What she said earlier made it seem like this was a monthly thing.

Reply:
It is, sort of. The event itself takes place many times per month, but it is generally considered to be something that a person only attends on roughly monthly basis. Enough of these take place in a month that all the Remnant present on that world would be able to attend. There's probably quite a few going on any particular day.

This is sort of like Friday religious services (or Saturday, or Sunday, depending on one's religion of choice); except that the cultural expectation is of monthly attendance, not weekly.

On a more subtle level, this is intended to show that yes, there is religion here; but it's much more casual and informal than many real-life present-day religious denominations.